src/platforms/python/guides/celery/index.mdx

```diff
@@ -32,8 +32,8 @@ To verify if your SDK is initialized on worker start, you can pass `debug=True`
 
 <Alert level="info" title="Note on distributed tracing">
 
-Sentry uses custom message headers for distributed tracing. Since Celery version 4.0, for [message protocol of version 1](https://docs.celeryproject.org/en/stable/internals/protocol.html#version-1), this functionality is broken and Celery fails to propagate custom headers to the worker. Protocol version 2, which is default since Celery version 4.0, is not affected.
+Sentry uses custom message headers for distributed tracing. For Celery versions 4.x, for [message protocol of version 1](https://docs.celeryproject.org/en/stable/internals/protocol.html#version-1), this functionality is broken and Celery fails to propagate custom headers to the worker. Protocol version 2, which is default since Celery version 4.0, is not affected.
```

Comment:
Trying to avoid the double "for" here. Do "with" or "using" still make sense?
```suggestion
Sentry uses custom message headers for distributed tracing. For Celery versions 4.x, with [message protocol of version 1](https://docs.celeryproject.org/en/stable/internals/protocol.html#version-1), this functionality is broken, and Celery fails to propagate custom headers to the worker. Protocol version 2, which is the default since Celery version 4.0, is not affected.
```